app/src/main/java/org/p2p/wallet/auth/ui/phone/PhoneNumberEnterFragment.kt

```diff
@@ -0,0 +1,89 @@
+package org.p2p.wallet.auth.ui.phone
+
+import android.os.Bundle
+import android.view.View
+import org.koin.android.ext.android.inject
+import org.p2p.wallet.R
+import org.p2p.wallet.auth.ui.phone.countrypicker.CountryPickerDialog
+import org.p2p.wallet.auth.model.CountryCode
+import org.p2p.wallet.common.mvp.BaseMvpFragment
+import org.p2p.wallet.databinding.FragmentAddNumberBinding
+import org.p2p.wallet.utils.viewbinding.getColor
+import org.p2p.wallet.utils.viewbinding.viewBinding
+
+class PhoneNumberEnterFragment :
+    BaseMvpFragment<PhoneNumberEnterContract.View, PhoneNumberEnterContract.Presenter>(R.layout.fragment_add_number),
+    PhoneNumberEnterContract.View {
+
+    companion object {
+        const val REQUEST_KEY = "REQUEST_KEY"
+        const val RESULT_KEY = "RESULT_KEY"
+        fun create() = PhoneNumberEnterFragment()
+    }
+
+    override val presenter: PhoneNumberEnterContract.Presenter by inject()
+    private val binding: FragmentAddNumberBinding by viewBinding()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) = with(binding) {
+        super.onViewCreated(view, savedInstanceState)
+        setOnResultListener()
+        presenter.load()
```

Comment:
i think this logic should be called in presenter in attachView{} or init {}

what do you think ?